modules/dnn/src/dnn_utils.cpp

```diff
@@ -382,6 +382,80 @@ void imagesFromBlob(const cv::Mat& blob_, OutputArrayOfArrays images_)
     }
 }
 
+Rect blobRectToImage(Rect r, InputArray oriImage, Image2BlobParams param)
+{
+    CV_Assert(!oriImage.getMat_().empty());
+    Size imgSize = oriImage.getMat_().size();
```

Comment:
Seems like image is not needed at all but only it's Size

Reply:
Agree my idea was : user gives same parameter than in blobFromImage called but it can be changed